youtubechat/ytchat.py

```diff
@@ -138,6 +140,8 @@ def permaban(self):
                                    'POST',
                                    headers={'Content-Type': 'application/json; charset=UTF-8'},
                                    body=jsondump)
+        popo = json.dumps(data)
```

Comment:
please rename this variable to 'returnjson' or something similar. you can also just do dumps(data) instead of json.dumps(data)